resources/language/resource.language.sv_se/strings.po

```diff
@@ -56,7 +56,7 @@ msgstr "Retrospect"
 
 msgctxt "#30009"
 msgid "General"
-msgstr "Allmän"
+msgstr "Allmänt"
```

Comment:
This now means "Generally", not "General"

Reply:
Even in Kodi they use `Allmänt`

![image](https://user-images.githubusercontent.com/5789283/99158534-00023900-26d4-11eb-9dcd-68c59481005d.png)

Reply:
Even Windows use `Allmänt`

![image](https://user-images.githubusercontent.com/5789283/99158667-9b47de00-26d5-11eb-830e-c0804aae62be.png)